ExternalHelpers/WindowsDarkMode.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Win32;
+
+namespace ExternalHelpers
+{
+    public class WindowsDarkMode
+    {
+        private static System.Windows.Forms.Timer timer = new System.Windows.Forms.Timer()
+        {
+            Interval = 1500
+        };
+
+        public static bool IsDarkModeEnabled { get; private set; }
+        public static event Action<bool>? DarkModeChanged;
```

Comment:
We seem to not use it. So, probably we can get rid of this completely, and just leave `IsDarkModeEnabled`:

```csharp
timer.Tick += delegate { IsDarkModeEnabled = FetchIsDarkModeEnabled(); };
```